components/match2/wikis/hearthstone/match_summary.lua

```diff
@@ -151,14 +151,14 @@ end
 ---@return Html?
 function CustomMatchSummary._displayOpponents(isTeamMatch, players, flip)
 	local playerDisplays = Array.map(players, function (player)
-		local char = HtmlWidgets.Div{
+		local char = player.class and HtmlWidgets.Div{
```

Comment:
possibly empty check instead of nil check?

Reply:
We can do it jus to be sure :D